### PR TITLE
[ADF-2765] Fix for the gallery view in datatable component

### PR DESCRIPTION
**Pull request by davidcanonieto**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Gallery view breaks flex layout at the bottom. (Last row).

**What is the new behaviour?**
Now every single row is displayed where it belongs.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-2765

### CODE REVIEW

File: lib/core/datatable/components/datatable/datatable.component.ts [hunk 1]
```diff
@@ -686,4 +695,16 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
             this.dataRowsChanged = null;
         }
     }
+
+    datatableLayoutFix() {
+        const maxGalleryRows = 7;
```
Comment (eromano) [inline lib/core/datatable/components/datatable/datatable.component.ts]: is this a magic number?

Comment (davidcanonieto) [reply lib/core/datatable/components/datatable/datatable.component.ts]: A little bit, it has to be as big as the number of maximum columns that are going to be displayed. We can add a comment or think of another solution.

Comment (eromano) [reply lib/core/datatable/components/datatable/datatable.component.ts]: this number I guess works only for a certain display size

**Comment by eromano** [comment]
ERROR: /home/travis/build/Alfresco/alfresco-ng2-components/lib/core/datatable/components/datatable/datatable.component.ts[163, 1]: trailing whitespace

**Pull request merged**